$$
\newcommand{pdiff}[3]{\left(\frac{\partial #1}{\partial #2}\right)_{#3}}
\newcommand{diff}[3]{\left(\frac{d #1}{d #2}\right)_{#3}}
\newcommand{partialT}[1]{\frac{\partial}{\partial T} \left(#1\right)_{N,V}}
$$

## Chem 576 HW2

### Exercise 3.3
Use Eq. (3.43) to show that the electronic partition function in Eq. (3.41) contributes only a constant to the energy, and thus makes a vanishing contribution to the constant-volume heat capacity.

------------

According to Eq.(3.41),
$$(p.f.)_{el} = e^{-\epsilon_0/k_B T} $$
Substitude in Eq.(3.43),
$$
\begin{align}
A^{el} &= -k_B T \ln{(e^{-\epsilon_0/k_B T})^N} \\
    &= -Nk_BT\left(-\frac{\epsilon_0}{k_BT}\right) \\
    &= N\epsilon_0 \\
E^{el} &= -T^2 \partialT{\frac{A^{el}}{T}} \\
    &= -T^2 \partialT{\frac{N\epsilon_0}{T}} \\
    &= -T^2N\epsilon_0\left(-\frac{1}{T^2}\right)\\
    &= N\epsilon_0\\
C_V &= \pdiff{E^{el}}{T}{N,V} = 0
\end{align}
$$
Therefore, electronic partition function only contributes $N\epsilon_0$ to the energy, and the contribution vanishes on constant-volume heat capacity $C_V$.

### Excercise 3.4
Show from Eqs.(3.43)–(3.44) that every mode in the internal partition function that has $(p.f.)_{mode} \propto T^\nu$ in the high-temperature limit, will contribute $\nu R$ to the molar constant-volume heat capacity.

--------------

Since 
$$(p.f.)_{mode} \propto T^\nu$$
any arbituary partition function of a mode can be expressed as 
$$(p.f.)_{mode} = \left(\lambda T^\nu\right)_{V,N}$$
where $\lambda$ is an arbituary constant.

Under constant $V$ and $N$, Helmholtz free energy as a function of $T$ can be expressed
$$
\begin{align}
\Big(A(T)\Big)_{V,N} &= -k_B T \ln{\left(\lambda T^\nu\right)^N} \\
    &= -\nu Nk_BT \ln{(\lambda T)} \\
    &= -\nu Nk_BT (\ln{T} + \ln{\lambda}) \\
\end{align}
$$
Total energy can be expressed as
$$
\begin{align}
E &= -T^2 \partialT{\frac{A_{mode}}{T}} \\
    &= -T^2 \frac{\partial}{\partial T}\Big(-\nu Nk_B(\ln{T} + \ln{\lambda})\Big)_{N,V} \\
    &= \nu Nk_BT^2 \frac{d}{d T}\big(\ln{T} + \ln{\lambda}\big) \\
    &= \nu Nk_BT^2\left(\frac{d \ln{T}}{d T}+0\right)\\
    &= \nu Nk_BT\\
C_V &= \pdiff{E}{T}{N,V} = \pdiff{(\nu Nk_BT)}{T}{N,V} \\
    &= \nu Nk_B
\end{align}
$$

For molar constant-volume heat capacity $C_{m,V}$,
$$N = N_A$$
Thus,
$$C_{m,V} = \nu N_A k_B = \nu R$$

### Exercise 3.8
The high-temperature limit of the rotational partition function for the homonuclear diatomic molecule in Eq.(3.55) contains a factor of $\rho^2$ for the nuclear spin degeneracy, where $\rho = 2I + 1$. Work through the corresponding Helmholtz free energy.

**(a)**
What is the effect of this degeneracy factor on the energy $E$? On the heat capacity $C_V$?

**(b)**
What is the effect of this factor on the entropy $S$? On the chemical potential $\mu$?

--------------

**(a)**

For heteronuclear diatomic molecules, the Helmholtz free energy is expressed as
$$
\begin{align}
A_{hetero}(T) &= -k_B T \ln{\left(\sum_l (2l+1)e^{-l(l+1)\theta_R/T}\right)^N} \\
    &= -Nk_B T \ln{\Big(\int_0^\infty e^{\theta_Ru/T}du}\Big) \\
    &= -Nk_BT \ln{\frac{T}{\theta_R}} \\
E_{hetero} &= -T^2 \partialT{\frac{A_{hetero}}{T}}\\
    &= -T^2 \partialT{-Nk_B \ln{\frac{T}{\theta_R}}}\\
    &= Nk_BT^2 \frac{d}{dT} \big(\ln{T}- \ln{\theta_R} \big)\\
    &= Nk_BT \\
C_{V,hetero} &= \pdiff{(Nk_BT)}{T}{N,V} \\
    &= Nk_B
\end{align}
$$
For homonuclear diatomic molecules, 
$$
\begin{align}
A_{homo}(T) &= -k_B T \ln{\left(\frac{1}{2}\rho^2 \sum_l (2l+1)e^{-l(l+1)\theta_R/T}\right)^N} \\
    &= -Nk_B T \ln{\Big(\frac{\rho^2}{2}\int_0^\infty e^{\theta_Ru/T}du}\Big) \\
    &= -Nk_BT \ln{\frac{\rho^2 T}{2\theta_R}} \\
E_{homo} &= -T^2 \partialT{\frac{A_{homo}}{T}}\\
    &= -T^2 \partialT{-Nk_B \ln{\frac{\rho^2 T}{2\theta_R}}}\\
    &= Nk_BT^2 \frac{d}{dT} \big(\ln{T} +2\ln{\rho} -\ln{2\theta_R} \big)\\
    &= Nk_BT \\
C_{V,homo} &= \pdiff{(Nk_BT)}{T}{N,V} \\
    &= Nk_B
\end{align}
$$
Therefore, the degeneracy has no effect on energy $E$ or heat capacity $C_V$

**(b)**

For entropy, we have
$$
\begin{align}
S_{hetero} &= -\pdiff{A_{hetero}}{T}{V,N}\\
    &= -\frac{\partial}{\partial T}\left( -Nk_BT\ln{\frac{T}{\theta_R}}\right)_{V,N}\\
    &= Nk_B\ln{\frac{T}{\theta_R}}+Nk_BT{\frac{1}{T}}\\
    &= Nk_B\left(1+\ln{\frac{T}{\theta_R}}\right)\\
S_{homo} &= -\pdiff{A_{homo}}{T}{V,N}\\
    &= -\frac{\partial}{\partial T}\left( -Nk_BT\ln{\frac{\rho^2T}{2\theta_R}}\right)_{V,N}\\
    &= Nk_B\ln{\frac{\rho^2T}{2\theta_R}}+Nk_BT{\frac{1}{T}}\\
    &= Nk_B\left(1+\ln{\frac{\rho^2T}{2\theta_R}}\right)\\
    &= Nk_B\left(1+\ln{\frac{T}{\theta_R}} +\ln{\frac{\rho^2}{2}} \right)\\
\end{align}
$$
The difference of entropy $\Delta S$ due to the degeneracy is
$$
\Delta S = S_{homo}-S_{hetero} = Nk_B\ln{\frac{\rho^2}{2}}
$$
For chemical potential, we have
$$
\begin{align}
\mu_{hetero} &= \pdiff{A_{hetero}}{N}{V,T}\\
    &= -k_B T \ln{\frac{T}{\theta_R}} \\
\mu_{homo} &= \pdiff{A_{homo}}{N}{V,T}\\
    &= -k_B T \ln{\left(\frac{\rho^2T}{2\theta_R}\right)} \\
    &= -k_B T \left( \ln{\frac{T}{\theta_R}} + \ln{\frac{\rho^2}{2}}\right)\\
\end{align}
$$

The difference of chemical potential $\mu$ due to the degeneracy is
$$
\Delta \mu = \mu_{homo}-\mu_{hetero} = -k_BT\ln{\frac{\rho^2}{2}}
$$

### Exercise 3.19
Consider the following chemical reaction between perfect gases
$$
A_2 + B_2 <=> 2AB
$$
**(a)** Show the law of mass action and equilibrium constant in this case are given by
$$
\frac{N_{AA} N_{BB}}{[N_{AB}]^2} = \frac{(p.f.)_{AA}(p.f.)_{BB}}{[(p.f.)_{AB}]^2}\equiv K_{eq}(T)
$$

where $K_{eq}(T)$ is independent of $V$.

**(b)** Show the chemical potentials satisfy
$$
\mu_1+\mu_2 = 2\mu_{12}
$$

------------------------

**(a)**

We have constraints for energy E, numbers of molecules $N_{AA}$, $N_{BB}$, $N_{AB}$, and number of atoms $N_A$ and $N_B$
$$
\begin{align}
2N_{AA}+N_{AB}&=N_A \\
2N_{BB}+N_{AB}&=N_B \\
\sum_i n_i^a \epsilon_i^a + \sum_i n_i^b \epsilon_i^b +\sum_i n_i^{ab} \epsilon_i^{ab} &= E
\end{align}
$$

The total number of complexion is 
$$
\begin{align}
\Omega &= \sum_{N_{AA},N_{BB},N_{AB}} \frac{1}{N_{AA}!\;N_{BB}!\;N_{AB}!}\sum_{\{n\}}\frac{N_{AA}!\;N_{BB}!\;N_{AB}!}{\prod_{i}n_i^{a!}\prod_{i}n_i^{b!}\prod_{i}n_i^{ab!}} \\
    &=\sum_{N_{AA},N_{BB},N_{AB}} \Omega(N_{AA},N_{BB},N_{AB};E) \\
\end{align}
$$
Using stirling approximation, and at the saddle point, the variation is zero
$$
\begin{align}
\delta \ln{\Omega(N_{AA},N_{BB},N_{AB};E)} &= \delta N_{AA}\ln{(p.f.)_{AA}}+\delta N_{BB}\ln{(p.f.)_{BB}}+\delta N_{AB}\ln{(p.f.)_{AB}}\\
    &-\delta N_{AA}\ln{N_{AA}}-\delta N_{BB}\ln{N_{BB}}-\delta N_{AB}\ln{N_{AB}}\\
    &=0
\end{align}
$$

Define small variabtions on number of particles $\delta N_{AA}$, $\delta N_{BB}$, $\delta N_{AB}$. The variations at constant $E$ are subject to the constraint that the net change in partical numbers has to be zero in respect to the total number of atoms ($N_A$ and $N_B$ are fixed)
$$
\begin{align}
2\delta N_{AA}+\delta N_{AB} &=0 \\
2\delta N_{BB}+\delta N_{AB} &=0 \\
\end{align}
$$
Using Lagrange's method of undetermined multipliers, multiply the equation by a constant $\alpha$,
$$
\begin{align}
2\alpha_A\delta N_{AA}+\alpha_A\delta N_{AB}&=0 \\
2\alpha_B\delta N_{BB}+\alpha_B\delta N_{AB}&=0 \\
\end{align}
$$

Add the terms to the $\delta \ln{\Omega}$ we have 
$$
\begin{align}
\delta \ln{\Omega(N_{AA},N_{BB},N_{AB};E)} 
    = &\delta N_{AA}(\ln{(p.f.)_{AA}}- \ln{N_{AA}}+ 2\alpha_A)+ \\
       &\delta N_{BB}(\ln{(p.f.)_{BB}}- \ln{N_{BB}}+ 2\alpha_B)+ \\
       &\delta N_{AB}(\ln{(p.f.)_{AB}}- \ln{N_{AB}}+ \alpha_A + \alpha_B)\\
    = &0
\end{align}
$$
Since, by construction, the variations on number of particles $\delta N_{AA}$, $\delta N_{BB}$, $\delta N_{AB}$ are none zero, the only condition to satify the above equation is
$$
\begin{align}
\ln{(p.f.)_{AA}}- \ln{N_{AA}}+ 2\alpha_A &= 0 \\
\ln{(p.f.)_{BB}}- \ln{N_{BB}}+ 2\alpha_B &= 0 \\
\ln{(p.f.)_{AB}}- \ln{N_{AB}}+ \alpha_A + \alpha_B &= 0\\
\end{align}
$$

Define $\lambda$
$$
\begin{align}
e^{2\alpha_A} &\equiv \lambda_A \\
e^{2\alpha_B} &\equiv \lambda_B \\
\end{align}
$$
Take exponentiation of the equations
$$
\begin{align}
\ln{N_{AA}} &= \ln{(p.f.)_{AA}}+ 2\alpha_A \\
\ln{N_{BB}} &= \ln{(p.f.)_{BB}}+ 2\alpha_A \\
\ln{N_{AB}} &= \ln{(p.f.)_{AB}} + (\alpha_A + \alpha_B)\\
N_{AA} &= e^{2\alpha_A}(p.f.)_{AA}\\
N_{BB} &= e^{2\alpha_B}(p.f.)_{BB}\\
N_{AB} &= e^{\alpha_A+\alpha_B}(p.f.)_{AB}\\
\end{align}
$$

Therefore,
$$
N_{AA}N_{BB} = e^{2(\alpha_A+\alpha_B)}(p.f.)_{AA}(p.f.)_{BB}
$$

and
$$
\frac{N_{AA} N_{BB}}{[N_{AB}]^2} = \frac{(p.f.)_{AA}(p.f.)_{BB}}{[(p.f.)_{AB}]^2}\equiv K_{eq}(T)
$$

**(b)**

$$
\begin{align}
A(T,V,N_{AA},N_{BB},N_{AB})&= -k_BT\ln{\frac{[(p.f.)_{AA}]^{N_{AA}}[(p.f.)_{BB}]^{N_{BB}}[(p.f.)_{AB}]^{N_{AB}}}{N_{AA}!N_{BB}!N_{AB}!}} \\
\mu_1 &= \pdiff{A}{N_{AA}}{T,V,N_{BB},N_{AB}} \\
      &= -k_BT\ln{\frac{(p.f.)_{AA}}{N_{AA}}} \\
      &= 2\alpha_Ak_BT \\
\mu_2 &= \pdiff{A}{N_{BB}}{T,V,N_{AA},N_{AB}} \\
      &= -k_BT\ln{\frac{(p.f.)_{BB}}{N_{BB}}} \\
      &= 2\alpha_Bk_BT \\
\mu_{12} &= \pdiff{A}{N_{AB}}{T,V,N_{AA},N_{BB}} \\
      &= -k_BT\ln{\frac{(p.f.)_{AB}}{N_{AB}}} \\
      &= (\alpha_{A}+\alpha_{B})k_BT \\
\end{align}
$$
Therefore,
$$\mu_1+\mu_2 = 2\mu_{12}$$

### Exercise 3.20
**(a)** Use the result in Prob. 3.19, and find an expression for the equilibrium constant $K_{eq}(T)$ for the following reaction
$$H_2+I_2 <=> 2HI$$
in terms of the partition functions for these diatomic molecules.

**(b)** Compute $K_{eq}(T)$ at 500 K from the following table of constants

The quantities appearing in this table are

$D$ = dissociation energy in cm$^{-1}$ <br>
$\omega$ = oscillator angular frequency in cm$^{-1}$ <br>
$r$ = internuclear spacing in \r{A}

**(c)** Compare with the experimental value of the equilibrium constant at that temperature
$$\log_{10}{[K_{eq}]^{-1}}=2.078$$

---------------------------

**(a)**

For this reaction, the equilibrium constant can be expressed as
$$
\begin{align}
K_{eq} &= \frac{(p.f.)_{H_2}(p.f.)_{I_2}}{[(p.f.)_{HI}]^2} \\
(p.f.) &= (p.f.)^{trans}(p.f.)^{vib}(p.f.)^{rot}(p.f.)^{el}
\end{align}
$$

where
$$
(p.f.)^{trans} = V\left[\frac{2\pi (m_1+m_2)k_BT}{h}\right]^{\frac32}
$$

$$(p.f.)^{vib} = \frac{e^{-\theta_V/2T}}{1-e^{-\theta_V/T}} = \frac{e^{-h\nu/2k_BT}}{1-e^{-h\nu/k_BT}}$$

For high temperature limit for vibrational partition function
$$(p.f.)^{rot}=\frac{8\pi^2I_{12}k_BT}{\sigma_{12}h^2}$$

Since, for temperature $T = 500K$, almost all the electron population will be in the ground state, and the electron partition function can be approximated to
$$(p.f.)^{el} = e^{-\epsilon_0/k_BT}$$

Since the equilibrium constant is independent of volume $V$, the $V$ in the translational p.f. is a constant. We can cancel out all the constants in $K_{eq}$

$$
\begin{align}
K_{eq} &= \frac{(p.f.)^{trans}_{H_2}(p.f.)^{trans}_{I_2}}{(p.f.)^{trans}_{HI}} \cdot
         \frac{(p.f.)^{vib}_{H_2}(p.f.)^{vib}_{I_2}}{(p.f.)^{vib}_{HI}} \cdot
         \frac{(p.f.)^{rot}_{H_2}(p.f.)^{rot}_{I_2}}{(p.f.)^{rot}_{HI}} \cdot
         \frac{(p.f.)^{el}_{H_2}(p.f.)^{el}_{I_2}}{(p.f.)^{el}_{HI}} \\
       &= \left[\frac{4(m_{H}m_{I})}{(m_{H}+m_{I})^2}\right]^{\frac{3}{2}} \cdot
          \left[\frac{e^{-h\nu_{H_2}/2k_BT}}{1-e^{-h\nu_{H_2}/k_BT}} \cdot
          \frac{e^{-h\nu_{I_2}/2k_BT}}{1-e^{-h\nu_{I_2}/k_BT}} \cdot
          \left(\frac{1-e^{-h\nu_{HI}/k_BT}}{e^{-h\nu_{HI}/2k_BT}}\right)^2 \right]\cdot 
          \left[\frac{I_{H_2}I_{I_2}}{I_{HI}^2} \cdot
          \frac{\sigma_{HI}^2}{\sigma_{H_2}\sigma_{I_2}} \right]\cdot
          \Big[e^{(-\varepsilon_{0,H_2}-\varepsilon_{0,I_2}+2\varepsilon_{0,HI})/k_BT}\Big] \\
\end{align}
$$

#### For rotational and vibrational contributions ####
Since 
$$I = \mu r_0^2$$
and 
$$\mu = \frac{m_1m_2}{m_1+m_2}$$
The expression becomes
$$I = \frac{m_1m_2}{m_1+m_2}\cdot {r_{0,12}}^2$$

and
$$
\begin{align}
\frac{I_{H_2}I_{I_2}}{I_{HI}^2} &= \frac{m_Hm_I}{4}\cdot\left(\frac{m_H+m_I}{m_Hm_I}\right)^2 \cdot \frac{{r_{0,H_2}}^2{r_{0,I_2}}^2}{({r_{0,HI}}^2)^2} \\
    &= \frac{(m_H+m_I)^2}{4m_Hm_I} \cdot \frac{(r_{0,H_2}\cdot r_{0,I_2})^2}{{r_{0,HI}}^4}
\end{align}
$$

Combining the terms with the ones from translational p.f.
$$
\begin{align}
\left[\frac{4m_{H}m_{I}}{(m_{H}+m_{I})^2}\right]^\frac{3}{2}\cdot \frac{(m_H+m_I)^2}{4m_Hm_I} \cdot \frac{(r_{0,H_2}\cdot r_{0,I_2})^2}{{r_{0,HI}}^4} \\
= \frac{2(m_{H}m_{I})^\frac{1}{2}}{m_{H}+m_{I}}\cdot \frac{(r_{0,H_2}\cdot r_{0,I_2})^2}{{r_{0,HI}}^4}
\end{align}
$$

Since, for each type of molecule
$$r_0 \equiv r \text{ (intermolecular spacing)}$$
The final terms become
$$
\frac{2(m_{H}m_{I})^\frac{1}{2}}{m_{H}+m_{I}} \cdot 
\frac{(r_{H_2}r_{I_2})^2}{{r_{HI}}^4}
$$

#### For electron contributions ###
The bonding energies of the reaction,
<img src="./BondEnergies.svg"
     alt="Bonding Energies Graph"
     width="400" />

$$
\begin{align}
\Delta \varepsilon_{rxn} &= 2\varepsilon_{prod}-\varepsilon_{react} \\
    &= -2D_{0,HI} + D_{0,H_2} + D_{0,I_2} \\
    &= 2\varepsilon_{0,HI} - \varepsilon_{0,H_2} - \varepsilon_{0,I_2} \\
\end{align}
$$
Where $D_0$ is the bond dissociation energy in $\text{J}$

Converting $D$ in $\text{cm}^{-1}$ to $\text{J}$
$$D_0 = Dhc$$
$$2\varepsilon_{0,HI} - \varepsilon_{0,H_2} - \varepsilon_{0,I_2} = -(2D_{HI}-D_{H_2}-D_{I_2})hc$$
The final terms become
$$
e^{-(2D_{HI}-D_{H_2}-D_{I_2})hc/k_BT}
$$

#### For translational contributions
Converting $\omega$ in $\text{cm}^{-1}\text{s}^{-1}$ to $\nu$ in $\text{s}^{-1}$
$$\nu = \frac{\omega c}{2\pi} $$
and
$$h\nu = \hbar\omega c$$
The final terms become
$$
\frac{e^{-\hbar\omega_{H_2}c/2k_BT}}{1-e^{-\hbar\omega_{H_2}c/k_BT}} \cdot
\frac{e^{-\hbar\omega_{I_2}c/2k_BT}}{1-e^{-\hbar\omega_{I_2}c/k_BT}} \cdot
\left(\frac{1-e^{-\hbar\omega_{HI}c/k_BT}}{e^{-\hbar\omega_{HI}c/2k_BT}}\right)^2 
$$

#### The final expression ####
Therefore, the expression of $K_{eq}$ in the given $D$,$\omega$,$r$ parameters is
$$
K_{eq} = 
    \frac{e^{-\hbar\omega_{H_2}c/2k_BT}}{1-e^{-\hbar\omega_{H_2}c/k_BT}} \cdot
    \frac{e^{-\hbar\omega_{I_2}c/2k_BT}}{1-e^{-\hbar\omega_{I_2}c/k_BT}} \cdot
    \left(\frac{1-e^{-\hbar\omega_{HI}c/k_BT}}{e^{-\hbar\omega_{HI}c/2k_BT}}\right)^2 
\cdot 
    \frac{2(m_{H}m_{I})^\frac{1}{2}}{m_{H}+m_{I}} \cdot 
    \frac{(r_{H_2}r_{I_2})^2}{{r_{HI}}^4} 
\cdot 
    \frac{\sigma_{HI}^2}{\sigma_{H_2}\sigma_{I_2}} 
\cdot
    e^{-(2D_{HI}-D_{H_2}-D_{I_2})hc/k_BT}
$$

**(b)**

Plugging in the numbers

In [79]:
from math import exp, pi, log10
h = 6.62607e-34 # J*s
hbar = 1.05457e-34 # J*s
kB = 1.38065e-23 # J/K
T = 500 # K
c = 2.99792e8 # m/s

mH = 1.0079
mI = 126.9045

sigH2 = 2
sigI2 = 2
sigHI = 1

DH2 = 38436*100 # m^-1
DI2 = 12625*100 # m^-1
DHI = 25944*100 # m^-1 (It was 24,944 on the textbook)

omH2 = 4395.0*100  # m^-1
omI2 = 214.6*100   # m^-1
omHI = 2309.0*100  # m^-1

rH2 = 0.7417e-10 # m
rI2 = 2.667e-10  # m
rHI = 1.604e-10  # m

def v_exp(omega):
    return exp(-hbar*omega*c/(2*kB*T))/(1-exp(-hbar*omega*c/(kB*T)))
    
## vib
k_eq_t = v_exp(omH2)*v_exp(omI2)*v_exp(omHI)**(-2)
print("k_vib =", k_eq_t)

## trans and rot
k_eq_vr = 2*(mH*mI)**0.5/(mH+mI) * (rH2*rI2)**2/rHI**4 * sigHI**2/(sigH2*sigI2)
print("k_tr =", k_eq_vr)

## el
k_eq_el = exp(-(2*DHI-DH2-DI2)*h*c/(kB*T))
print("k_el =",k_eq_el)

## K_eq the equilibrium constant
k_eq = k_eq_t*k_eq_vr*k_eq_el
print("k_eq =", k_eqr
print("log10(1/k_eq) =", log10(1/k_eq))

k_vib = 5.262586790305365
k_tr = 0.02613302741781733
k_el = 0.09257542676023117
k_eq = 0.012731650791930576
log10(1/k_eq) = 1.8951152818651602


In [76]:
# This answer matches the one (energy in eV) in solution manual, 
# But the D_HI has changed to 25944
print(-(2*DHI-DH2-DI2)*h*c/(1.602e-19))

-0.10254607846085395


In [72]:
# This should've been the energy in eV using the number from textbook
# But it doesn't give the correct answer
print(-(2*24944-12625-38436)/8066)

0.145425241755517

**(c)**

In [78]:
print((2.078-1.895)/2.078)

0.08806544754571696


The $\log{(1/K)}$ value calcuated from the partion functions is $8.8\%$ lower than the experimental value 

### Exercise 3.21 ####
Consider all the participants in a chemical reaction to be perfect gases. Show that the volume dependence in the equilibrium constant $K(T,V)$ is just such as to allow one to re-express the law of mass action in terms of the particle densities $n_i = N_i/V$.

--------------------